modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -31,27 +31,38 @@ The `SMA2` connector is bidirectional.
 ====
 
 Set `spec.profile.plugins.e810.ublxCmds` parameters to configure the GNSS clock in the `PtpConfig` custom resource (CR).
+
+[IMPORTANT]
+====
+You must configure separate delay offset values to counteract 1PPS signal delay and T-GM antenna cable signal delay.
+To calculate optimal offset values that do not introduce constant time errors, make precise measurements of the 1PPS cable and GNSS antenna cable signal delays.
+Red Hat cannot assist in this measurement or provide any values for the required delay offsets.
+====
```

Comment:
A bit confused about our use of delay and offset together and interchangeably through the text. Can we use "offset" when we mean an adverse effect and "delay compensation" when we mean fixing  it?